packages/app/src/lib/workers/encoding.ts

```diff
@@ -327,6 +327,19 @@ export const eventVariantCodec = Kinds({
   "space.roomy.admin.remove.0": Struct({
     adminId: str,
   }),
+  /**
+   * This event sets the ATProto account did that should be used as the handle for this space.
+   *
+   * For this to be verified that account also has to have a `space.roomy.stream` PDS record with an
+   * rkey of `handle` and an `id` value that is set to this streams ID.
+   *
+   * When both the stream points to the ATProto account, and the ATProto account points to the
+   * stream ID, then that makes a verified Roomy space handle so you can visit the space at
+   * `https://roomy.space/example.handle`.
+   */
+  "space.roomy.stream.handle.account.0": Struct({
```

Comment:
I also added this event to allow the space to track which ATProto account it should use as it's handle. This is the other side of the 2-way verification:

1. You can resolve an ATProto handle to a space by reading the `space.roomy.stream/handle` record and getting the `id` field.
2. You can resolve a Roomy space to an ATProto handle by reading the stream handle account to get a DID and then fetching the ATProto handle for that DID.